### DISCUSSION TITLE
DevToys on winget package source winget

### DISCUSSION THREAD

**Discussion by Kronhjortens** [DevToys-app/DevToys#659] [Q&A] [answered]
Hi,
Could you place releases on winget package source winget?
Thanks :-)

**Comment by nickdevore** [comment]
I was able to install it from `winget` today without any problems.

**Reply by Kronhjortens** [reply]
It's correct you can install with source msstore.
But you can't install it with source winget.
![image](https://user-images.githubusercontent.com/59131719/193800635-aa08c0fd-c225-4708-88cf-40cab0fba9d7.png)

**Reply by nickdevore** [reply]
Ah, my bad.  I'm just curious, why is that important to you?  Assuming the source is a trusted source, what does it matter where it comes from?

**Comment by Kronhjortens** [comment]
msstore source doesn't include version, so  it's not possible to update
using Winget update Devtoys in a cmd and it's not possible to install
msstore on windows server 2022.
Winget source is more Linux 'apt get' like, which I find very nice to use.
:-)

**Comment by jogo-** [comment, marked as answer]
@veler 
Please update the latest (pre-)release version available via winget, since it's still the **2.0.5**, while the latest (pre-)release is now **2.0.7** :
```
C:\>winget list DevToys-app.DevToys
Nom                                   ID                  Version       Source
-------------------------------------------------------------------------------
DevToys Preview version 2.0-preview.5 DevToys-app.DevToys 2.0-preview.5 winget

```

**Reply by veler** [reply]
It's in progress, for 2.0.6.0 at least. https://github.com/microsoft/winget-pkgs/pull/190299

**Reply by Kronhjortens** [reply]
Thanks this makes it possible to update with a command line. :-)
